src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingService.cs

```diff
@@ -252,6 +265,7 @@ private async Task<ImmutableArray<TextChange>> ApplyFormattedChangesAsync(
         ImmutableArray<IFormattingPass> formattingPasses,
```

Comment:
Are there any overloads of this that take in multiple passes any more? Could potentially simplify?

Reply:
oh that's a good point!